buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -432,43 +474,31 @@ class AssemblyResourceGeneration extends DefaultTask {
         project.views = []
         project.partitions = []
         project.isAssembly = true
-        log.info('Creating visible resources')
-        project.visibleResources = components.findAll {
-            it.startsWith(VANTIQ_SYSTEM_PREFIX + VANTIQ_SERVICES)
-        }
-        log.info('finding source...')
-        def sourceName = components.find {
-            it.startsWith(VANTIQ_SYSTEM_PREFIX + VANTIQ_SOURCES)
-        }
-        log.info('Found projects source reference: {}', sourceName)
-        sourceName = sourceName.substring(sourceName.lastIndexOf('/') + 1)
-        log.info('Found projects source name {}', sourceName)
 
         project.options = [
             description: description,
             filterBitArray: 'ffffffffffffffffffffffffffffffff',
             type: 'dev',
             v: 5,
             isModeloProject: true,
```

Comment:
We create assemblies that are _layered on_ (have a dependency on or prerequisite of) the connector def'n assembly that includes the sourceimpl & schema type.  We don't want to delete those when this assembly is uninstalled, so exclude things things are that are added in via dependency processing.